blocktx/store/sql/GetTransactionBlock.go

```diff
@@ -25,9 +17,19 @@ func (s *SQL) GetTransactionBlock(ctx context.Context, transaction *blocktx_api.
 		WHERE t.hash = $1
 		AND b.orphanedyn = false
 	`
+)
+
+func (s *SQL) GetTransactionBlock(ctx context.Context, transaction *blocktx_api.Transaction) (*blocktx_api.Block, error) {
+	start := gocore.CurrentNanos()
+	defer func() {
+		gocore.NewStat("blocktx").NewStat("GetTransactionBlock").AddTime(start)
```

Comment:
I think you can just write `defer gocore.NewStat("blocktx").NewStat("GetTransactionBlock").AddTime(start)`

Reply:
I added another commit which fixes this.